procmon_test.go

```diff
@@ -36,9 +36,19 @@ func TestProcessMonitor_PrepareEvent(t *testing.T) {
 	fileEventPart3 := make(map[string]interface{})
 	fileEventPart3["cwd"] = "/dir"
 
+	fileEventPart4 := make(map[string]interface{})
+	fileEventPart4["argc"] = "2"
+	fileEventPart4["a0"] = "argv1"
+	fileEventPart4["a1"] = "argv2"
+
+	fileEventPart5 := make(map[string]interface{})
+	fileEventPart5["argc"] = []string{"failed"}
```

Comment:
what is being tested here? It would be good to start adding comments for what these tests are looking to test...

Reply:
@varunsh-coder thanks for pointing it out.. I had started adding comments for the same